.vscode/settings.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
im trialing using a `settings.json.default` with a few things like the ones in here in them for share around.
speaking of the code in here, i've submitted this: https://github.com/bazelbuild/vscode-bazel/issues/132

Reply:
i removed this because it seems like an error that you should be configuring my local vscode.
ill read the issue for background

Reply:
Yeah I think this should be ignored aswell.
It would be nice if we could check in some settings for the workspace, however vs code dosen't currently support having two settings files that get merged together.
That's why my work around is to make a default one that devs can use as a template